src/file_handle_cache.erl

```diff
@@ -449,7 +449,8 @@ needs_sync(Ref) ->
     %% This must *not* use with_handles/2; see bug 25052
     case get({Ref, fhc_handle}) of
         #handle { is_dirty = false, write_buffer = [] } -> false;
-        #handle {}                                      -> true
+        #handle {}                                      -> true;
+        undefined                                       -> true
```

Comment:
Why should the function return `true` if the handle is unknown?

Reply:
Sorry, meant to open this as a draft. It's not done!